README.md

```diff
@@ -1,29 +1,72 @@
 # Inspirehep
 
-### Installation
+## Installation
 
-#### pyenv (optional)
+### How to run
+
+These commands allow you to spin up a contenarized environment that reproduces the Inspire-HEP and Inspire-Next applications.
 
-install `pyenv` https://github.com/pyenv/pyenv
+#### With docker-compose
+
+##### Setup/Run:
 
 ```bash
-$ pip install pipenv
+$ docker-compose up -d
 ```
 
-### How to run
-
-These commands allow you to spin up a contenarized environment that reproduces the Inspire-HEP and Inspire-Next applications.
+##### How to test:
 
 ```bash
-$ ./scripts/bootstrap
 $ docker-compose up -d
-$ docker-compose run --rm web ./scripts/setup
-$ docker-compose run --rm web-next scripts/setup_inspire_next
+$ docker-compose run --rm web ./run-tests.sh
 ```
 
-### How to test
+#### Local machine
+
+You need to setup db, es, rabbit-mq etc by yourself (or you can use docker images)
+
+##### Setup:
+
+1.  Install poetry
+
+```bash
+$ curl -sSL https://raw.githubusercontent.com/sdispater/poetry/master/get-poetry.py | python
+```
```

Comment:
`To get started you need Poetry's bin directory ($HOME/.poetry/bin) in your `PATH`` ;)

Reply:
Right, I will update this ;)